src/core/factcheck_plus/__init__.py

```
@@ -0,0 +1,37 @@
+    "ROLE_TO_CLASS",
+    "SelectionResult",
+    "join_claims",
+    "PolicyScorer",
+    "load_policy",
+    "DEFAULT_POLICY_PATH",
+    "ARGUMENT_MAPPING_SCHEMA",
+    "REFINED_CLAIM_SCHEMA",
+    "RATING_DIMS",
+    "SchemaError",
+    "validate",
+]
```

Ruff (0.15.20)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)